test/arrays.coffee

```diff
@@ -74,8 +74,8 @@ test "array elisions destructuring with splats and expansions", ->
   arrayEq [a,b], [2,[5,6,7,8,9]]
   [,c,...,,d,,e] = arr
   arrayEq [c,d,e], [2,7,9]
-  [...,e,,,f,,,] = arr
-  arrayEq [e,f], [4,7]
+  [...,f,,,g,,,] = arr
+  arrayEq [f,g], [4,7]
```

Comment:
Why did this change?

Reply:
This test threw some errors, so I changed variables, as `e` was used in the line above, and I thought this was the reason (it wasn't).
I can change it back as it's not needed for this PR.

Reply:
It's better not to reuse variables. If it passes either way, let's keep the non-reused version.